.github/workflows/python-package.yaml

```diff
@@ -18,14 +18,24 @@ jobs:
       - name: Checkout
```

Comment:
Note that usually we want to install Python packages into the system library with `uv` by setting `UV_SYSTEM_PYTHON=1`. However, in this case, we want `uv` to install things into the tox-created virtualenv, so we leave that var unset.